spec/lib/vanagon/platform/windows_spec.rb

```diff
@@ -18,6 +31,12 @@
     context "on #{plat[:name]} we should behave ourselves" do
```

Comment:
@glennsarti We can certainly tidy this, but its not directly related to this set of changes.